source/NVDAObjects/UIA/__init__.py

```diff
@@ -256,94 +257,72 @@ def _getFormatFieldAtRange(  # noqa: C901
 				IDs.add(UIAHandler.UIA_AnnotationTypesAttributeId)
 			IDs.add(UIAHandler.UIA_CultureAttributeId)
 			fetcher = BulkUIATextRangeAttributeValueFetcher(textRange, IDs)
+		def fetch(id):
+			return fetcher.getValue(id, ignoreMixedValues=ignoreMixedValues)
 		if formatConfig["reportFontName"]:
-			val = fetcher.getValue(UIAHandler.UIA_FontNameAttributeId, ignoreMixedValues=ignoreMixedValues)
+			val = fetch(UIAHandler.UIA_FontNameAttributeId)
 			if val != UIAHandler.handler.reservedNotSupportedValue:
 				formatField["font-name"] = val
 		if formatConfig["reportFontSize"]:
-			val = fetcher.getValue(UIAHandler.UIA_FontSizeAttributeId, ignoreMixedValues=ignoreMixedValues)
+			val = fetch(UIAHandler.UIA_FontSizeAttributeId)
 			if isinstance(val, numbers.Number):
 				# Translators: Abbreviation for points, a measurement of font size.
 				formatField["font-size"] = pgettext("font size", "%s pt") % float(val)
```

Comment:
If I were to refactor this, instead I would create helper functions for each of these if blocks, particularly the longer ones.
The aliasing of a function is still possibly worth while, but splitting up the function based on if/else logic will have a bigger impact.

@coderabbitai, can you suggest a refactor for the function getFormatFieldAtRange

Reply:
can you move the helper functions outside of `getFormatFieldAtRange`? Create parameters to pass in any required arguments

Reply:
Thanks!  I agree that approach looks much nicer.  I'll update the  PR with this.

Do you think that it would be worth moving the helper functions into a class or something?  It bothers me a little to clutter up the UIATextInfo class with so many helpers.

BTW, I had also experimented with a more aggressive approach that got rid of the need for ID preregistration (which is duplicative and error-prone).  The idea was to create a list of "handlers", where each handler was a pair of (attribute_id, thunk).  Unfortunately there were a couple places where that approach required fairly tricky code, due to interaction between more than one attribute id.  Similar to your suggestion here, that approach also required creating a bunch of helper functions.